fastai/basic_data.py

```diff
@@ -116,7 +116,7 @@ def create(cls, train_ds:Dataset, valid_ds:Dataset, test_ds:Optional[Dataset]=No
         datasets = cls._init_ds(train_ds, valid_ds, test_ds)
         val_bs = ifnone(val_bs, bs)
         dls = [DataLoader(d, b, shuffle=s, drop_last=s, num_workers=num_workers, **dl_kwargs) for d,b,s in
-               zip(datasets, (bs,val_bs,val_bs,val_bs), (True,False,False,False)) if d is not None]
+               zip(datasets, (bs,val_bs,val_bs,val_bs), (False,False,False,False)) if d is not None]
```

Comment:
I think we want to keep shuffling train in other experiments, so we should ideally pass this down.

Reply:
Ya makes sense, will make the change